java/se/sics/mspsim/chip/CC1101.java

```diff
@@ -239,118 +239,89 @@ public void log(String str) {
-            txNext();
-            setState(CC1101RadioState.CC1101_STATE_TX);
-                        break;
-
-                case CC1101_SIDLE:
-                        setState(CC1101RadioState.CC1101_STATE_IDLE);
-                        currentRssiValid = false;
-                        break;
-
-                case CC1101_SAFC:
-                        log("CC1101_SAFC not implemented");
-                        break;
-
-                case CC1101_SWOR:
-                        log("CC1101_SWOR not implemented");
-                        break;
-
-                case CC1101_SPWD:
-                    //			log("CC1101_SPWD almost implemented");
-                        /* TODO XXX
-                        * Wait until CS is de-asserted. (We should at least wait
-                        * receiving or transmitting.)*/
-                        setState(CC1101RadioState.CC1101_STATE_SLEEP);
-                        break;
-
-                case CC1101_SFRX:
-                        rxfifo.clear();
-                        rxExpectedLen = -1;
-                        rxGotSynchByte = false;
-                        setGDO0(false);
-                        /*printRXFIFO();*/
-                        break;
-
-                case CC1101_SFTX:
-                        txfifo.clear();
-                        txFooterCountdown = -1;
-                        /*printTXFIFO();*/
-                        break;
-
-                case CC1101_SWORRST:
-                        log("CC1101_SWORRST not implemented");
-                        break;
-
-                case CC1101_SNOP:
-                    //			log("CC1101_SNOP not implemented");
-                        break;
-
-                default:
-                        System.out.printf("strobe(0x%02x)\n", cmd);
-                        break;
-                }
+            case CC1101_SIDLE -> {
+              setState(CC1101RadioState.CC1101_STATE_IDLE);
+              currentRssiValid = false;
+            }
+            case CC1101_SAFC -> log("CC1101_SAFC not implemented");
+            case CC1101_SWOR -> log("CC1101_SWOR not implemented");
+            case CC1101_SPWD ->
+              //			log("CC1101_SPWD almost implemented");
+              /* TODO XXX
+             * Wait until CS is de-asserted. (We should at least wait
+             * receiving or transmitting.)*/
+                    setState(CC1101RadioState.CC1101_STATE_SLEEP);
+            case CC1101_SFRX -> {
+              rxfifo.clear();
+              rxExpectedLen = -1;
+              rxGotSynchByte = false;
+              setGDO0(false);
+              /*printRXFIFO();*/
+            }
+            case CC1101_SFTX -> {
+              txfifo.clear();
+              txFooterCountdown = -1;
+              /*printTXFIFO();*/
+            }
+            case CC1101_SWORRST -> log("CC1101_SWORRST not implemented");
+            case CC1101_SNOP -> {
+              //			log("CC1101_SNOP not implemented");
```

Comment:
Same about removing most of the leading spaces in this comment, and same for the one on line 302.